sdk/communication/communication-identity/test/communicationIdentityClient.spec.ts

```diff
@@ -28,9 +28,10 @@ describe("CommunicationIdentityClient [Playback/Live]", function() {
   });
 
   it("successfully creates a user and token", async function() {
-    const { user, token } = await client.createUserWithToken(["voip"]);
-    assert.isString(user.communicationUserId);
-    assert.isString(token);
+    const userWithToken = await client.createUserWithToken(["voip"]);
```

Comment:
I prefer destructuring, was the linter complaining about it?

Reply:
Similar answer here. Linter was complaining `user` was a shadowed variable. I am not sure if destructing is the more preferred syntax in TS?